Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM ubuntu:latest
+FROM fedora:30
 MAINTAINER Mansi Kulkarni "mankulka@redhat.com"
 RUN apt-get update -y
 RUN apt-get install -y python3-pip python3-dev build-essential
```

Comment:
I belive apt-get does not work in fedora systems . 
please change the command to its equivalent dnf and change the package names too. 
ie., python3-dev changes to python3-devel

Please try testing the Dockerfile by installing docker on your machine and run
```
sudo docker build . 
```
to see if its working or not.